noxfile.py

```diff
@@ -86,7 +85,7 @@ def lint(session):
     Returns a failure if the linters find linting errors or sufficiently
     serious code quality issues.
     """
-    session.install(FLAKE8_VERSION, BLACK_VERSION)
```

Comment:
also this. So we pin on 6.1.0.

The source template was updated as well.
https://github.com/googleapis/synthtool/pull/1842

Reply:
Fixed, all these changes were done automatically with owlbot. Updated to new version of owlbot to sync with synthtool so should be ok now.